### ISSUE TITLE
Problem with 2019.1.1 in Visual Studio 2019

### ISSUE THREAD

**Issue by Mike-E-angelo** [matkoch/resharper-testlinker#14]
I have installed this as an extension in R# 2019.1.1 in Visual Studio 2019, and have mapped the key to go to linked items.  However, when I attempt to execute it, it doesn't work at all, unfortunately.

Is this a known issue, perhaps?  I am posting here so that it will naturally start to work like magic only after I have posted a question online about it. 😆

**Comment by Mike-E-angelo** [comment]
Related: https://youtrack.jetbrains.com/issue/RSRP-474619

**Comment by matkoch** [comment]
Do you get an exception reported? Might be necessary to start Visual Studio with /Resharper.Internal switch.

**Comment by Mike-E-angelo** [comment]
Exception?  If you mean a toast or notification of some sort, no I do not.  The keypress simply does not appear to work for either go-to-linked-files or rename with the extension installed.  Uninstalling the extension returns renaming functionality, but I no longer have the extension. 😁

I would appreciate further guidance on the `/Resharper.Internal` switch and what to look for upon using it.

**Comment by matkoch** [comment]
You just start devenv.exe with that as an argument. If an exception occurs, it will be shown with a red icon in the lower right statusbar.

**Comment by Mike-E-angelo** [comment]
Cool!  Like EAPs but not.  Yes, I had a couple exceptions occur from opening the solution with this switch enabled.

Here is what I see from this extension now:

```
Component descriptor TestLinker.LinkedTypesProvider.NameSuffixLinkedTypesProvider [Singleton, Corrupted] is corrupted and cannot be accessed

--- EXCEPTION #1/2 [InvalidOperationException]
Message = “Component descriptor TestLinker.LinkedTypesProvider.NameSuffixLinkedTypesProvider [Singleton, Corrupted] is corrupted and cannot be accessed”
ExceptionPath = Root.InnerException
ClassName = System.InvalidOperationException
Data.ThreadLocalDebugInfo = ScheduleNextTask
HResult = COR_E_INVALIDOPERATION=80131509
Source = JetBrains.Platform.ComponentModel
StackTraceString = “
  at JetBrains.Application.Components.SingletonDescriptor.GetValue()
     at JetBrains.Application.Components.EnumerableValueResolver.EnumerableComponentDescriptor`1.<GetEnumerator>d__5.MoveNext()
”

--- Outer ---

--- EXCEPTION #2/2 [LoggerException]
Message = “Component descriptor TestLinker.LinkedTypesProvider.NameSuffixLinkedTypesProvider [Singleton, Corrupted] is corrupted and cannot be accessed”
ExceptionPath = Root
ClassName = JetBrains.Util.LoggerException
Data.ManagedThreadName = <NULL>
Data.SccRevisionShell = “<there are no packages matching the criteria>”
Data.HostProductInfo = “JetBrains ReSharper Ultimate 2019.1.1 Build 191.0.20190501.122851”
Data.SubProducts.#0 = “JetBrains dotCover 2019.1.1 Build 2019.1.20190501.125303”
Data.SubProducts.#1 = “JetBrains dotTrace 2019.1.1 Build 2019.1.20190501.125303”
Data.SubProducts.#2 = “JetBrains ReSharper 2019.1.1 Build 2019.1.20190501.124844”
Data.SccRevisionEnv = “
  Platform\Core\Shell:
  	git::refs/heads/191-eap9-rtm-2019.1.1
  
  Platform\VisualStudio:
  	git::refs/heads/191-eap9-rtm-2019.1.1::2804164ae647a7995195eb3dbaaa971ca020cf07
”
Data.VsVersion = 16.0.28803.352
InnerException = “Exception #1 at Root.InnerException”
HResult = COR_E_APPLICATION=80131600
StackTraceString = “
  at JetBrains.Application.Components.EnumerableValueResolver.EnumerableComponentDescriptor`1.<GetEnumerator>d__5.MoveNext()
     at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
     at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
     at TestLinker.LinkedTypesService..ctor(LinkedNamesCache linkedNamesCache, IEnumerable`1 linkedTypesProviders, IUnitTestElementStuff unitTestElementStuff)
     at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
     at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
     at JetBrains.Application.Extensibility.PartCatalogTypeComponentDescriptor.CreateInstanceOf(Type type, IValueResolveContext context)
     at JetBrains.Application.Extensibility.PartCatalogTypeComponentDescriptor.CreateInstance(IValueResolveContext context)
     at JetBrains.Application.Components.SingletonDescriptor.CreateInstanceChecked(OnError onError, IComponentContainer container)
     at JetBrains.Application.Components.SingletonDescriptor.GetValue()
     at JetBrains.Application.Components.InitializationStrategyDefault.Schedule(Lifetime lifetime, ICollection`1 dscs, IComponentContainer container)
     at JetBrains.Application.Components.ComponentStorage.ComposeDescriptors(ICollection`1 descriptors)
     at JetBrains.Application.Components.ComponentStorage.Compose()
     at JetBrains.ProjectModel.SolutionInstance.OpenSolution(ISolutionOwner solutionOwner, ISolutionLoadTasksScheduler taskScheduler)
     at JetBrains.ProjectModel.SolutionManagerBase.<>c__DisplayClass23_0.<CreateSolutionInstance>b__0()
     at JetBrains.ProjectModel.Tasks.SolutionLoadTasksSchedulerBase.<>c__DisplayClass21_0.<ExecuteTask>b__1()
     at JetBrains.Util.ILoggerEx.Catch(ILogger thіs, Action F, ExceptionOrigin origin, LoggingLevel loggingLevel)
     at JetBrains.ProjectModel.Tasks.SolutionLoadTasksSchedulerBase.ExecuteTask(SolutionLoadTask task)
     at JetBrains.ProjectModel.Tasks.SolutionLoadTasksSchedulerAsync.<Schedule>b__5_0()
     at JetBrains.Application.Threading.IShellLocksEx.ExecuteWithReadLock(IShellLocks thіs, Action F)
     at JetBrains.Application.Threading.IShellLocksEx.<>c__DisplayClass11_0.<QueueReadLock>b__0()
     at JetBrains.Threading.ReentrancyGuardEx.<>c__DisplayClass0_1.<Queue>b__3()
     at JetBrains.Threading.ReentrancyGuard.Execute(String name, Action action)
     at JetBrains.Threading.ReentrancyGuard.ExecutePendingActions()
     at JetBrains.Threading.JetDispatcher.Closure.Execute()
     at JetBrains.Util.Concurrency.WinJetDispatcher.ProcessQueue(Int32 nMinBucket)
     at JetBrains.Util.Concurrency.WinJetDispatcher.<>c__DisplayClass18_0.<BeginInvokeCore>b__0()
     at System.Windows.Threading.DispatcherOperation.InvokeDelegateCore()
     at System.Windows.Threading.DispatcherOperation.InvokeImpl()
     at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
     at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
     at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
     at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
     at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
     at System.Windows.Threading.DispatcherOperation.Invoke()
     at System.Windows.Threading.Dispatcher.ProcessQueue()
     at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
     at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
     at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
     at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
     at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
     at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
     at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
”

```

**Comment by Mike-E-angelo** [comment]
I've also submitted via the report tool.  Here are a few:
https://youtrack.jetbrains.com/issue/DEXP-441941
https://youtrack.jetbrains.com/issue/DEXP-451620
https://youtrack.jetbrains.com/issue/DEXP-451595
https://youtrack.jetbrains.com/issue/DEXP-451601

Some of these are reporting 404s but not sure if that's because they are set to private on your side or not.

**Comment by Mike-E-angelo** [comment]
OK digging around this a little more, there appears to be an installation error that occurs with this extension in 2019.1.1.  

I had another issue (https://youtrack.jetbrains.com/issue/RSRP-474612) where five installation errors occurred.  Turns out these were caused by Enhanced Tooltip, which I installed along with this (and many others 😁) on the first try.  Uninstalling all extensions and installing Test Linker by itself produces the following (single) error (which was probably hidden in/masked by the other five, previously):

```
1 error occurred.

17:00:27.886 |E| Error resolving type JetBrains.DataFlow.Lifetime from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False!

--- EXCEPTION #1/1 [LoggerException]
Message = “Error resolving type JetBrains.DataFlow.Lifetime from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False!”
ExceptionPath = Root
ClassName = JetBrains.Util.LoggerException
Data.ManagedThreadName = <NULL>
HResult = COR_E_APPLICATION=80131600
StackTraceString = “
  Error resolving type JetBrains.DataFlow.Lifetime from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False!
  
  --- EXCEPTION #1/2 [TypeLoadException]
  Message = “Error resolving type JetBrains.DataFlow.Lifetime from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False!”
  ExceptionPath = Root.InnerException
  ClassName = System.TypeLoadException
  HResult = COR_E_TYPELOAD=80131522
  TypeLoadResourceID = 0
  
  --- Outer ---
  
  --- EXCEPTION #2/2 [LoggerException]
  Message = “Error resolving type JetBrains.DataFlow.Lifetime from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”, request originating from assembly “JetBrains.Platform.RdCore, Version=777.0.0.0, Culture=neutral, PublicKeyToken=1010a0d8d6380325”. Could not find the type by its FQN. Is assembly qualification specified? True! Is it of our own assembly? True! Is looking into referenced assemblies allowed? False! Was the qualified referenced assembly encountered during lookup? False! Tried loading an implicitly-referenced assembly? False!”
  ExceptionPath = Root
  ClassName = JetBrains.Util.LoggerException
  InnerException = “Exception #1 at Root.InnerException”
  HResult = COR_E_APPLICATION=80131600
  StackTraceString = “
    at JetBrains.Util.OnError.<>c__DisplayClass17_0.<LogForeignException>b__0(Exception ex)
       at JetBrains.DataFlow.Signal`1.NotifySinks(TValue payload)
       at JetBrains.DataFlow.Signal`1.Fire(TValue value, Object cookie)
       at JetBrains.DataFlow.Signal`1.Fire(TValue value)
       at JetBrains.Metadata.Reader.Impl.MetadataAssembly.GetTypeInfoFromQualifiedName(String name, AssemblyNameInfo assemblyName, Boolean searchReferencedAssemblies)
       at JetBrains.Metadata.Reader.Impl.MetadataAssembly.GetTypeInfoFromToken(MetadataToken token)
       at JetBrains.Metadata.Reader.Impl.MetadataAssembly.GetTypeFromToken(MetadataToken token, TypeDecodeContext context)
       at JetBrains.Metadata.Reader.Impl.MetadataAssembly.GetTypeFromToken(MetadataToken token)
       at JetBrains.Metadata.Reader.Impl.MetadataTypeBuilder.BuildClassType(MetadataToken typeDefOrRefOrSpecToken)
       at JetBrains.Metadata.Reader.Impl.MethodSignatureBlobDecoder.DecodeBlob[TType,TSig](IBinaryReader reader, IMethodSignatureBuilder`2 builder)
       at JetBrains.Metadata.Reader.Impl.MetadataMethod.get_Signature()
       at JetBrains.Metadata.Reader.Impl.MetadataMethod.get_ReturnValue()
       at JetBrains.Application.Table.Emit.EmitCatalogTables.Types.GetTypeDeclarationMembers(TypeRef typeref, IMetadataTypeInfo type, EmitTimeTables tables)
       at JetBrains.Application.Table.Emit.EmitCatalogTables.Types.FillPreCreatedShallowTypeDeclaration(TypeRef typeref, IMetadataTypeInfo type, EmitTimeTables tables)
       at JetBrains.Application.Table.Emit.EmitCatalogTables.Types.FillPreCreatedShallowTypes(EmitTimeTables tables)
       at JetBrains.Application.Table.Emit.EmitCatalogTables.<>c__DisplayClass1_0.<CreateCatalogTablesWithMetadataReader>b__0(Lifetime lifetime)
       at JetBrains.Lifetimes.Lifetime.Using(Action`1 action)
       at JetBrains.Application.Table.Emit.EmitCatalogTables.CreateCatalogTablesWithMetadataReader(IEnumerable`1 assemblies, IAssemblyMasterBinder masterbinder, ICollection`1 traitcalc, Stream output)
       at JetBrains.Application.BuildScript.Application.Catalogs.PartCatalogs.CreateWithMetadataReaderRaw(Lifetime lifetime, IEnumerable`1 assemblies, IAssemblyMasterBinder masterbinder)
       at JetBrains.Application.Install.ProductPartCatalogFromPackages.GetProductPartsCatalogFromProductAssemblies(Lifetime lifetime, InputPackage[] ipkgs, IBuildTimeRealizedAssemblyItemUnification realizedAssemblyItemUnification, AllAssembliesOnEverything allass, IBuildTaskHost tasker, ILogger logger)
       at JetBrains.Application.Install.ProductPartCatalogFromPackages..ctor(Lifetime l, InputPackage[] b, IBuildTimeRealizedAssemblyItemUnification d, AllAssembliesOnEverything f, IBuildTaskHost g, ILogger e)
       at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
       at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
       at JetBrains.Build.Running.Storage.BuildHelper.JetBrains.Build.Running.Storage.IBuildHelper.CreateInstance(IList`1 inputsChosen, Object[] inputvalues)
       at JetBrains.Build.Running.Storage.BuildStorageInstances.<>c__DisplayClass6_0.<CreateEntityInstance_Helper>b__0(LifetimeDefinition definition)
       at JetBrains.Lifetimes.Lifetime.Define(Lifetime lifetime, String id, Action`1 atomicAction)
       at JetBrains.Build.Running.Storage.BuildStorageInstances.CreateEntityInstance_Helper(BuildStorageCore storage, IBuildHelper helper, OnError onerror)
       at JetBrains.Build.Running.Storage.BuildStorageInstances.CreateEntityInstance(BuildStorageCore storage, IBuildEntity entity, OnError onerror)
       at JetBrains.Build.Running.Logic.BuildExecutor.<ExecuteSingleItemAsync>d__16.MoveNext()
       at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
       at JetBrains.Build.Running.Logic.BuildExecutor.ExecuteSingleItemAsync(Lifetime lifetime, PlannedItem item, BuildStorage storage, ITaskHost tasker, ILogger Logger)
       at JetBrains.Build.Running.Logic.BuildExecutor.<>c__DisplayClass14_1.<<RunAsync>b__7>d.MoveNext()
       at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
       at JetBrains.Build.Running.Logic.BuildExecutor.<>c__DisplayClass14_1.<RunAsync>b__7(Lifetime lifetimeStep)
       at JetBrains.Lifetimes.Lifetime.<UsingNestedAsync>d__48.MoveNext()
       at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine](TStateMachine& stateMachine)
       at JetBrains.Lifetimes.Lifetime.UsingNestedAsync(Func`2 action)
       at JetBrains.Build.Running.Logic.BuildExecutor.<>c__DisplayClass14_1.<<RunAsync>b__4>d.MoveNext()
       at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Runtime.CompilerServices.AsyncMethodBuilderCore.MoveNextRunner.Run()
       at JetBrains.Application.Threading.Tasks.TaskHost.AccessViolationCatcher(Action action)
       at JetBrains.Application.Threading.Tasks.TaskHost.<>c__DisplayClass39_0.<Create>b__1(Object state)
       at System.Threading.Tasks.Task.Execute()
       at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
       at System.Threading.Tasks.Task.ExecuteEntry(Boolean bPreventDoubleExecution)
       at JetBrains.Application.Threading.Tasks.Scheduler.JetScheduler.ExecuteTask(Task task)
       at JetBrains.Application.Threading.Tasks.Scheduler.JetSchedulerThread.EnqueueNextTask()
       at JetBrains.Application.Threading.Tasks.Scheduler.JetSchedulerThread.ThreadPoolProc()
       at JetBrains.Util.Reflection.CallStackAnnotation.InvokeAnnotated(String classNameOfNewFrame, String methodNameOfNewFrame, Action actionToAnnotate)
       at JetBrains.Util.Reflection.CallStackAnnotation.CatchAnnotatedInvocation[TClassOfNewFrame](String methodNameOfNewFrame, Action actionToAnnotate)
       at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
       at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
       at System.Threading.ThreadHelper.ThreadStart()
  ”
”


TECHNICAL INFO
Microsoft Windows NT 10.0.17763.0 (Windows 10 Pro, version 6.3, build 17763, release 1809)
Time zone: (UTC-05:00) Eastern Time (US & Canada)
Input languages: en-US (US). Default: en-US (US)
Visual Studios
  15.0_a5b94cd5: Visual Studio Community 2017, Version 15.9.28307.586 (Nickname="" Preview=False Incomplete=False)
  16.0_7dd4fe16: Visual Studio Community 2019, Version 16.0.28803.352 (Nickname="" Preview=False Incomplete=False)
Installer
  Wave: 191
  Identity: 191.0.20190501.132557
  Gallery: Local disk package gallery
  Per-machine: False
  Run as administrator: True
  Raw command line: /HostsToReInstall=ReSharperPlatformVs16_7dd4fe16
  Parsed options: /HostsToReInstall=ReSharperPlatformVs16_7dd4fe16
Products root: C:\Users\micha\AppData\Local\JetBrains (free 72025 MB of 135719 MB)
Products before install
  dotCover191 (Wave=191)
    dotCover 2019.1.1 (191.0.20190501.125303)
  dotMemory191 (Wave=191)
    dotMemory 2019.1.1 (191.0.20190501.125302)
  dotPeek191 (Wave=191)
    dotPeek 2019.1.1 (191.0.20190501.124838)
  dotTrace191 (Wave=191)
    dotTrace 2019.1.1 (191.0.20190501.125303)
  ReSharperPlatformVs16_7dd4fe16 (Wave=191 VS=16.0 Id=7dd4fe16)
    ReSharper 2019.1.1 (191.0.20190501.124844)
    dotMemory (191.0.20190501.125302)
    dotTrace (191.0.20190501.125303)
    dotCover (191.0.20190501.125303)
    Humanizer Annotations 5.0.0
    JetBrains XunitTemplates 1.1.0
    Prodot ReCommendedExtension 3.4.4
Install targets
  update  ReSharperPlatformVs16_7dd4fe16 at C:\Users\micha\AppData\Local\JetBrains\Installations\ReSharperPlatformVs16_7dd4fe16_001
Products after install
  dotCover191 (Wave=191)
    dotCover 2019.1.1 (191.0.20190501.125303)
  dotMemory191 (Wave=191)
    dotMemory 2019.1.1 (191.0.20190501.125302)
  dotPeek191 (Wave=191)
    dotPeek 2019.1.1 (191.0.20190501.124838)
  dotTrace191 (Wave=191)
    dotTrace 2019.1.1 (191.0.20190501.125303)
  ReSharperPlatformVs16_7dd4fe16 (Wave=191 VS=16.0 Id=7dd4fe16)
    ReSharper 2019.1.1 (191.0.20190501.124844)
    dotMemory (191.0.20190501.125302)
    dotTrace (191.0.20190501.125303)
    dotCover (191.0.20190501.125303)
    Humanizer Annotations 5.0.0
    JetBrains XunitTemplates 1.1.0
    Prodot ReCommendedExtension 3.4.4
    ReSharper TestLinker 2018.3.0

```

**Comment by matkoch** [comment]
Unfortunately, I can't reproduce this issue. Everything is working fine, including navigation of course.

<img width="941" alt="image" src="https://user-images.githubusercontent.com/5005566/57227683-48866400-7012-11e9-89d3-27db3b858957.png">

**Comment by Mike-E-angelo** [comment]
Fortunately 2018.3.4 installs on VS2019 so I am good again. :)

This might be due to having Visual Studio 2017 installed in conjunction with Visual Studio 2019.  I will follow up with RSRP-474612 and close this issue since it does not appear to be due to this extension.

Thanks again for your time and assistance!

**Comment by SomeBeavers** [comment]
Hi @matkoch. The issue with Rename (https://youtrack.jetbrains.com/issue/RSRP-474619) is valid for ReSharper 2019.1.1 and TestLinker 2018.3.0.

[Error after extension is installed](https://github.com/matkoch/resharper-testlinker/files/3173582/Gohuhez.txt)
[Steps recording](https://github.com/matkoch/resharper-testlinker/files/3173597/Recording.253.zip)
[Test solution](https://github.com/matkoch/resharper-testlinker/files/3173599/MvvmBlazor-master_1.zip)

**Comment by Mike-E-angelo** [comment]
Does that mean we should reopen this issue? 😁😇

**Comment by matkoch** [comment]
I published a new version already. Should be fixed.

**Comment by Mike-E-angelo** [comment]
oooooh... ok cool!  I guess this means I can start braving back into 2019.1.1 again. 👍

**Comment by Mike-E-angelo** [comment]
Good news bad news... good news: I upgraded to 2019.1.1 and this issue is indeed now resolved.  THANK YOU!  Bad news: is that heap viewer is yet not ready (https://github.com/controlflow/resharper-heapview/issues/61) so it's back to 2018.3.5 for now.
